Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1363,6 +1363,54 @@ def is_tag_list_valid(tag_ids):
             raise DemistoException(f"Tag id has to be a positive integer, please change the given: '{tag}' id.")
 
 
+def create_updated_attribute_instance(demisto_args, attribute_uuid):
+    attribute_type = demisto_args.get('type')
+    distribution = demisto_args.get('distribution')
+    category = demisto_args.get('category')
+    comment = demisto_args.get('comment')
+    value = demisto_args.get('value')
+    first_seen = demisto_args.get('first_seen')
+    last_seen = demisto_args.get('last_seen')
+
+    attribute_instance = MISPAttribute()
+    attribute_instance.uuid = attribute_uuid
+    if attribute_type:
+        attribute_instance.type = attribute_type
+    if distribution:
+        attribute_instance.distribution = MISP_DISTRIBUTION_TO_IDS[distribution]
+    if category:
+        attribute_instance.category = category
+    if value:
+        attribute_instance.value = value
+    if comment:
+        attribute_instance.comment = comment
+    if first_seen:
+        attribute_instance.first_seen = first_seen
+    if last_seen:
+        attribute_instance.last_seen = last_seen
+    return attribute_instance
+
+
+def update_attribute_command(demisto_args: dict = {}):
```

Comment:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/
```suggestion
def update_attribute_command(demisto_args) -> CommandResults:
```

Reply:
Thanks, done.